BLSQ/iaso
# IA-2622 /  IA-2623: Org units change requests instances details and  files

Display reference instance content while reviewing org unit change request.

Document / images/video component is outdated too.

We should also be able to see the list of files on compare instances tools and change request review

Related JIRA tickets : IA-2622, IA-2623

## Self proofreading checklist


## Changes

- Move all components used to display instances files to TSX
- Review lay-out of instance details with a custom accordion fro Mui5
- display files component on instance details components
- fix video and image lay-out broken since Mui 5

## How to test

- first you need to linked files to an instance, edit one instance on the admin side and add images, videos, pdf, ...
- make sure you can see files in the list view of instances or by comparing this instance to another one
- add this instances to the "New reference instances" of an org unit change request in the admin, check the dialog, you should see the detail of the submission and files
## Print screen / video


https://github.com/BLSQ/iaso/assets/12494624/2d32b74a-8026-4ba2-87c2-39fd2abf6541


<img width="1604" alt="Screenshot 2024-01-22 at 15 41 22" src="https://github.com/BLSQ/iaso/assets/12494624/79773223-20cc-4410-a06a-5ddd946c8adb">
<img width="1648" alt="Screenshot 2024-01-22 at 15 41 07" src="https://github.com/BLSQ/iaso/assets/12494624/b2fae560-90ee-4170-878f-bee46a06c373">
